src/components/commanderOverview/CommanderDetails.tsx

```diff
@@ -48,10 +49,13 @@ export const CommanderDetails = React.memo(function CommanderDetails() {
     const matches = useSelector((state: AppState) =>
```

Comment:
Can we provide a comment here and/or  where we defined "matchesFilteredByPlayerCount" explaining why we are using 2 different match collections?

Reply:
Sure, done.